scripts/in_container/run_prepare_provider_packages.sh

```diff
@@ -24,7 +24,7 @@ LIST_OF_DIRS_FILE=$(mktemp)
 
 cd "${AIRFLOW_SOURCES}/airflow/providers" || exit 1
 
-find . -type d | sed 's/.\///; s/\//\./g' | grep -E 'hooks|operators|sensors|secrets|utils' \
+find . -type d | sed 's!./!!; s!/!.!g' | grep -E 'hooks|operators|sensors|secrets|utils' \
```

Comment:
Slight tidy up -- can use different character than `/` for regex in sed, meaning `\/` isn't needed when dealing with paths/urls.

Reply:
Here I removed "./" in front and then replaced  "/" with . to turn directories into providers "apache/hive" -> "apache.hive". That was the original meaning of this sed. I don't think the new sed does the same.